package.json

```diff
@@ -1,17 +1,34 @@
 {
-  "name": "",
-  "version": "",
+  "name": "buy-sell",
+  "version": "1.0.0",
   "description": "",
-  "author": "",
+  "author": "Chris Caden",
   "license": "",
   "engines": {
     "node": ">=6.13.0"
   },
   "scripts": {
-  },
-  "dependencies": {
+    "react-dev": "webpack -d --watch",
+    "server-dev": "nodemon server/index.js"
   },
   "devDependencies": {
-    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor"
+    "babel-core": "^6.23.1",
+    "babel-loader": "^6.3.2",
+    "babel-preset-es2015": "^6.22.0",
```

Comment:
use `babel-preset-env`, not `babel-preset-es2015` https://babeljs.io/docs/en/babel-preset-es2015

Reply:
updated